cpp/oneapi/dal/algo/dbscan/backend/gpu/kernel_fp_impl.hpp

```diff
@@ -39,59 +39,87 @@ inline std::int64_t get_recommended_check_block_size(const sycl::queue& queue,
     return 1;
 }
 
+inline std::int64_t get_recommended_min_width(const sycl::queue& queue,
+                                              std::int64_t column_count = 0,
+                                              std::int64_t wg_size = 0) {
+    //This min_width values has been computed via experiments,
+    //it shows the number of columns where the subgroups usage is effective
+    return 4;
+}
+
+///  A struct that finds the core points via subgroups
+///
+/// @tparam Float Floating-point type used to perform computations
+/// @tparam use_weights bool type used to check that weights are enabled
+///
+/// @param[in]  queue The SYCL queue
+/// @param[in]  data  The input data of size `row_count` x `column_count`
+/// @param[in]  weights  The input weights of size `row_count` x `1`
+/// @param[in]  cores  The current cores of size `row_count` x `1`
+/// @param[in]  neighbours  The current neighbours of size `row_count` x `1`
+/// it contains the counter of neighbours for each point
+/// @param[in]  epsilon  The input parameter epsilon
+/// @param[in]  min_observations  The input parameter min_observation
+/// @param[in]  deps  Events indicating availability of the `data` for reading or writing
```

Comment:
Please align the descriptions of the parameters as it is done in the rest of the codebase.
The goal is to make the code more unified. It should look like everything was written by a single person.

```suggestion
/// @tparam Float        Floating-point type used to perform computations
/// @tparam use_weights  Bool type used to check that weights are enabled
///
/// @param[in]  queue            The SYCL queue
/// @param[in]  data             The input data of size `row_count` x `column_count`
/// @param[in]  weights          The input weights of size `row_count` x `1`
/// @param[in]  cores            The current cores of size `row_count` x `1`
/// @param[in]  neighbours       The current neighbours of size `row_count` x `1`
///                              it contains the counter of neighbours for each point
/// @param[in]  epsilon          The input parameter epsilon
/// @param[in]  min_observations The input parameter min_observation
/// @param[in]  deps             Events indicating availability of the `data` for reading or writing
```